docs/code-search/code-navigation/how-to/index_a_go_repository.mdx

```diff
@@ -100,27 +100,27 @@ The following projects have example Travis CI configurations to generate and upl
 
 1. Install the [Sourcegraph CLI](https://github.com/sourcegraph/src-cli) with the following command:
 
-   ```
+```sh
    curl -L https://sourcegraph.com/.api/src-cli/src_linux_amd64 -o /usr/local/bin/src
    chmod +x /usr/local/bin/src
-   ```
+```
```

Comment:
This code block is meant to be part of the list item, so the triple backticks should be indented as earlier; same point applies later in this file.